meshroom/ui/app.py

```diff
@@ -95,6 +96,13 @@ def __init__(self, args):
         # Request any potential computation to stop on exit
         self.aboutToQuit.connect(r.stopExecution)
 
+        parser = argparse.ArgumentParser(prog=args[0], description='Launch Meshroom UI.')
+        parser.add_argument('--pipeline', metavar='MESHROOM_FILE', type=str, required=False,
```

Comment:
Maybe `project` would be more relevant here.

Reply:
ok, i just thought to be consistent with the other binary and effectively project is more self-explanatory.

Reply:
Yes I understand :) In the other binary the `pipeline` argument is to load a custom template pipeline (uninitialized project with custom parameters/nodes) and run it on input images, while in this case I would say it is mainly to load an initiliazed project file.